src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/custom-timing-details.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Use $har for custom timing details
+tags:
+    - Synthetics
+    - Synthetic monitoring
+    - Scripting monitors
+    - Scripted API monitor
+metaDescription: You can now use $har to have your scripted API monitors report custom timing details that appear in the New Relic UI.
+redirects:
+---
+
+<Callout variant="important">
+You need [the new runtime](/docs/synthetics/synthetic-monitoring/getting-started/new-runtime) to retrieve new custom timing details from your scripted API monitors. 
+</Callout>
+
+$har is a library that gives you access to both [default node.js module](/docs/synthetics/synthetic-monitoring/scripting-monitors/import-nodejs-modules/#supported-core-modules) and [importable node.js modules](/docs/synthetics/synthetic-monitoring/using-monitors/manage-monitor-runtimes/#v0). These modules simplify:
+
+* Creating timers to test DNS and FTP requests that report to the New Relic UI.
+* Creating `SyntheticRequest` events for your scripted API monitors. 
+
+## Set up your custom timers
+
+Your timing details will appear in the New Relic UI as a waterfall view. To get those details, create your timing resources using `addResource()` or `getResource()`. You start these timers by using `startTimer()`. Likewise, you stop these timers by using `stopTimer()`. 
```

Comment:
Based on the comment above suggesting changes to previous wording, I think we can remove the "Your timing details will appear in the New Relic UI as a waterfall view" reference here. We could then start this section with:

"Create timing resources using `addResource()` or `getResource(). You start these timers by using `startTimer()`. Likewise, you stop these timers by using `stopTimer()`."